styles/styles.scss

```diff
@@ -3,32 +3,69 @@
   --navbar-link-color: rgba(51, 51, 51, 0.7);
   --navbar-link-hover-color: #333333;
   --navbar-link-active-color: #333333;
+  --text-color-primary: #5f5e5e;
+  --primary-color: rgb(54, 120, 244);
   --spacer-smaller: 0.5rem;
   --spacer-small: 0.75rem;
   --spacer-default: 1rem;
   --spacer-big: 1.25rem;
   --spacer-bigger: 1.5rem;
   --spacer-biggest: 2.5rem;
   --font-size-normal: 1.125rem;
+  --font-weight-normal: 500;
   --font-weight-bold: 700;
 }
 
 body {
   font-family: 'Mulish', sans-serif;
   background-color: var(--navbar-bg-color);
+  margin: var(--spacer-biggest);
 }
 
+.img-content {
+  text-align: center;
+}
 
-.button {
+.img-content img {
+  max-width: 90%;
+}
 
+.text-content {
+  display: flex;
+  width: 47%;
+  flex-direction: column;
+  text-align: left;
+  position: relative;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -70%)
+}
+
+.text-content p {
```

Comment:
Mamy tu scss, więc można zrobić

`.text-content {

    p { }
}`

Reply:
próbowałem tak robić, ale gubiłem kolor czcionki, ale moge pogrzebać jescze nad tym

Reply:
To w następnym już